Signal.xcodeproj/project.pbxproj

```diff
@@ -2010,7 +2009,11 @@
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				34C6B0AE1FA0E4AA00D35993 /* test-jpg.jpg in Resources */,
```

Comment:
Maybe for another day, but we should bundle these in debug builds for sending fake attachments through the debug ui.

Reply:
Unfortunately, I don't know of a way to do this without creating a separate target which will be a pain.